ui/shared/status/StatusContextMenuButton.qml

```diff
@@ -0,0 +1,44 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQml 2.14
+import QtGraphicalEffects 1.13
+import "../../imports"
+import "../../shared"
+
+RoundButton {
+    id: control
+    implicitWidth: 32
+    implicitHeight: 32
+    contentItem: Item {
+        anchors.fill: parent
+
+        SVGImage {
+            id: iconImg
+            source: "/../../app/img/dots-icon.svg"
+            width: 18
+            height: 4
+            anchors.horizontalCenter: parent.horizontalCenter
+            anchors.verticalCenter: parent.verticalCenter
+            fillMode: Image.PreserveAspectFit
+        }
+
+        ColorOverlay {
+            id: iconColorOverlay
+            anchors.fill: iconImg
+            source: iconImg
+            color: Style.current.contextMenuButtonForegroundColor
+            antialiasing: true
+        }
+    }
+    background: Rectangle {
+        radius: Style.current.radius
+        color: hovered ? Utils.setColorAlpha(Style.current.black, 0.1) : Style.current.transparent
```

Comment:
I think there should be a better color for the hover, maybe we have to add it as a new theme color, but at least we'd have better control over it

Reply:
As per design, the hover color is 10% black, so this should be correct. But I'm happy to move it into a theme color for now, then we can always adjust it.